src/Framework/Framework/Configuration/DotvvmExperimentalFeaturesConfiguration.cs

```diff
@@ -17,11 +17,15 @@ public class DotvvmExperimentalFeaturesConfiguration
         [JsonProperty("explicitAssemblyLoading")]
         public DotvvmGlobalFeatureFlag ExplicitAssemblyLoading { get; private set; } = new DotvvmGlobalFeatureFlag();
 
+        [JsonProperty("useDotvvmSerializationForStaticCommandArguments")]
+        public DotvvmGlobalFeatureFlag UseDotvvmSerializationForStaticCommandArguments { get; private set; } = new DotvvmGlobalFeatureFlag();
```

Comment:
Rename to `UseDotvvmSerializationForStaticCommands`
Add XML comment
Will be true by default in the next release
Make sure it applies also for static command response